integration-tests/it-producer/src/main/java/org/apache/servicecomb/it/schema/TransportPojoSchema.java

```diff
@@ -30,4 +30,14 @@
   public String checkTransport(HttpServletRequest request) {
     return request instanceof InvocationToHttpServletRequest ? Const.HIGHWAY : Const.RESTFUL;
   }
+
+  //string
+
+  public String stringBody(String input) {
```

Comment:
This two methods never invoked in this PR

Reply:
sorry, there is some thing wrong when I merge the new updates, I forget to delete it